notebook.ipynb

```diff
@@ -2,9 +2,17 @@
  "cells": [
```

Comment:
<p>As a consequence of the wildcard import statements mentioned in my comment above, I cannot really follow here where the individual functions you are calling come from. I would also suggest to add some more context so that the function calls are understable without consulting the original paper.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eds-book-gallery/reproduce-deep-prior-4Dvar/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>@crina16 I've added some definition of the acronyms and symbols. We'll open an issue to indicate further comments or context of the steps would be beneficial to improve the readiness of the cell.</p>

Reply:
closing conversation, resolved